src/base.jl

```diff
@@ -1991,6 +1991,39 @@ function convert_component!(
     return
 end
 
+"""
+Converts a PowerLoad component to a StandardLoad component and replaces the original in the
+system. Does not set any fields in StandardLoad that lack a PowerLoad equivalent
+"""
+function convert_component!(
+    new_type::Type{StandardLoad},
+    old_load::PowerLoad,
+    sys::System;
+    kwargs...,
```

Comment:
In Julia the argument after the `!` is the one being modified. In this case is the system that is being changed. 

```suggestion
function convert_component!(
    sys::System,
    old_load::PowerLoad
    new_type::Type{StandardLoad},
    ;
    kwargs...,
```

Reply:
The existing `convert_component!` methods have the new type as the first argument, then the old component, then the system: https://github.com/NREL-Sienna/PowerSystems.jl/blob/1b7c1c0c4c65a11808640de2b556157a062bd2ee/src/base.jl#L1922-L1927 and https://github.com/NREL-Sienna/PowerSystems.jl/blob/1b7c1c0c4c65a11808640de2b556157a062bd2ee/src/base.jl#L1956-L1961 and I was maintaining consistency with those. Should I change those existing methods to follow the Julia convention?

Reply:
Yes, I should. Fixed in https://github.com/NREL-Sienna/PowerSystems.jl/pull/1022/commits/703762a15651373fff3ddf1905aad34917b30934.